api/client/service.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package nighthawk.client;
+
+import "api/client/options.proto";
+import "api/client/output.proto";
+
+message SendCommandRequest {
+  enum CommandType {
+    kStart = 0;
+    kUpdate = 1;
+    UNSET = 2;
+  }
+  CommandType command_type = 1;
+  CommandLineOptions options = 2;
```

Comment:
Do you want to use a `oneof` to model the different command options here? I'm thinking that `CommandLineOptions` would only apply to start, whereas other options would later apply to update..

Reply:
Maybe one way to offer the update call could be to also take a CommandLineOptions, and then we emit an error when a change is attempted that could not be processed on the fly? Regardless the `oneof` allows for more flexibility - added `oneof`.